src/main/java/tech/gruppone/stalker/server/configuration/SecurityConfiguration.java

```diff
@@ -1,8 +1,13 @@
 package tech.gruppone.stalker.server.configuration;
 
-// @EnableWebFluxSecurity
-// @EnableReactiveMethodSecurity
-// @PropertySource("classpath:application.properties")
-class SecurityConfiguration {
-  // TODO configure spring security here
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@PropertySource("classpath:application.properties")
+@Configuration
+public class SecurityConfiguration {
+
+  protected void configure() {
+    return;
+  }
```

Comment:
non mi è chiaro perché qui ci vada questo metodo?

Reply:
Ah mi sono dimenticato di toglierlo. In origine, il corpo di questo metodo diceva in quali endpoint far apparire il form per l'autenticazione LDAP. Questo però è inutile, in quanto il form è già stato costruito graficamente nella mobile application e non è Spring che deve occuparsi di farlo apparire. Lo tolgo.